src/main/java/ch/njol/skript/expressions/ExprInverse.java

```diff
@@ -0,0 +1,64 @@
+package ch.njol.skript.expressions;
+
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+
+@Name("Inverse Boolean")
+@Description("An expression to obtain the inverse value of a boolean")
+@Examples("set {_gravity} to inverse of player's flight mode")
+@Since("INSERT VERSION")
+public class ExprInverse extends SimpleExpression<Boolean> {
+
+	static {
+		Skript.registerExpression(ExprInverse.class, Boolean.class, ExpressionType.COMBINED, 
+			"[the] (inverse|opposite)[s] of %booleans%"
+		);
+	}
+
+	private Expression<Boolean> booleanExpr;
+
+	@Override
+	@SuppressWarnings("unchecked")
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
+		booleanExpr = (Expression<Boolean>) exprs[0];
+		return true;
+	}
+	
+	@Override
+	@Nullable
+	protected Boolean[] get(Event event) {
```

Comment:
```suggestion
	protected Boolean @Nullable [] get(Event event) {
```

Reply:
Do you confirm me the ``Nullable`` annotation is between the brackets and ``Boolean``? Not ``Boolean[] @Nullable``?

Reply:
yes the array is nullable, not the contents of the array
so the annotation should act on [], not on Boolean.